cli/local/validations.go

```diff
@@ -168,7 +168,9 @@ func ValidateLocalAPIs(apis []userconfig.API, projectFiles ProjectFiles, awsClie
 
 		pulledThisImage, err := docker.PullImage(image, dockerAuth, docker.PrintDots)
 		if err != nil {
-			return errors.Wrap(err, "failed to pull image", image)
+			if err := docker.CheckLocalImageAccessible(dockerClient, image); err != nil {
```

Comment:
Looking at the implementation for `PullImage()`, it seems that it doesn't return an error if the image already exists locally (however it may be missing the `latest` tag check). If that's the case, we should be able to add the latest tag check and revert this file; do you think that's a good idea?

Reply:
Yes, that makes sense. I went with using `CheckLocalImageAccessible` inside `docker.PullImage`.